develop-docs/application-architecture/dynamic-sampling/fidelity-and-biases.mdx

```diff
@@ -71,30 +72,19 @@ An example of how the UI looks is shown in the following screenshot (the content
 
 ![Biases in the UI](./images/biasesUI.png)
 
-### Deprioritize Health Checks
 
-This bias is used to de-prioritize transactions that are classified as health checks. The goal is to reduce the amount of data retained for health checks, since they are not very useful for debugging.
 
-In order to mark a transaction as a health check, we leverage a list of known health check endpoints, which is maintained by Sentry and updated regularly.
+### Prioritize New Releases
 
-```python
-HEALTH_CHECK_GLOBS = [
-    "*healthcheck*",
-    "*healthy*",
-    "*live*",
-    "*ready*",
-    "*heartbeat*",
-    "*/health",
-    "*/healthz",
-    # ...
-]
-```
+This bias is used to prioritize traces that are coming from a new release. The goal is to increase the sample rate in the time window that occurs between the creation of a release and its adoption by users. _The identification of a new release is done in the `event_manager` defined [here](https://github.com/getsentry/sentry/blob/master/src/sentry/event_manager.py#L937-L937)._
```

Comment:
The line numbers are likely to move frequently. What would you prefer to make this a stable reference?
- Pin a current revision
- Reference some name from code
- Move the code to a more stable place and reference the file

Reply:
Pinned it - after I had to search for it because it had indeed moved a lot 😅